src/components/com_tjdashboard/site/languages/en-GB/en-GB.com_tjdashboard.ini

```diff
@@ -23,59 +23,57 @@ COM_TJDASHBOARD_ERROR_MESSAGE_NOT_AUTHORISED = "You don't have enough permission
-
-
-;Widget View
-COM_TJDASHBOARD_TITLE_WIDGET="Dashboard Widgets"
+COM_TJDASHBOARD_TITLE_DASHBOARD="Dashboard"
+COM_TJDASHBOARD_LEGEND_DASHBOARD="Dashboard"
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_ID="ID"
+COM_TJDASHBOARD_FORM_DESC_DASHBOARD_ID=""
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_ORDERING="Order"
+COM_TJDASHBOARD_FORM_DESC_DASHBOARD_ORDERING=""
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_STATE="State"
+COM_TJDASHBOARD_FORM_DESC_DASHBOARD_STATE=""
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_CHECKED_OUT="N/A"
+COM_TJDASHBOARD_FORM_DESC_DASHBOARD_CHECKED_OUT=""
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_CHECKED_OUT_TIME="N/A"
+COM_TJDASHBOARD_FORM_DESC_DASHBOARD_CHECKED_OUT_TIME=""
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_CREATED_BY="Created by"
+COM_TJDASHBOARD_FORM_DESC_DASHBOARD_CREATED_BY=""
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_MODIFIED_BY="Modified by"
+COM_TJDASHBOARD_FORM_DESC_DASHBOARD_MODIFIED_BY=""
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_TITLE="Title"
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_CREATED_ON="Title"
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_CONTEXT="Context"
+COM_TJDASHBOARD_FORM_DESC_DASHBOARD_TITLE=""
+COM_TJDASHBOARD_TITLE_LIST_VIEW_DASHBOARDS="Dashboards"
+COM_TJDASHBOARD_TITLE_LIST_VIEW_DASHBOARDS_DESC="Show a list of Dashboards"
+COM_TJDASHBOARD_DASHBOARDS_ACTIONS="Actions"
+COM_TJDASHBOARD_DASHBOARDS_EDIT="Edit"
+COM_TJDASHBOARD_DASHBOARDS_DELETE="Delete"
+COM_TJDASHBOARD_TITLE_ITEM_VIEW_DASHBOARD="Single Dashboard"
+COM_TJDASHBOARD_TITLE_ITEM_VIEW_DASHBOARD_DESC="Show a specific Dashboard"
+COM_TJDASHBOARD_TITLE_FORM_VIEW_DASHBOARD="DashboardForm"
+COM_TJDASHBOARD_TITLE_FORM_VIEW_DASHBOARD_DESC="Show a form to add or edit a Dashboard"
+
+;Widget View
+COM_TJDASHBOARD_TITLE_WIDGET="Dashboard Widgets"
 COM_TJDASHBOARD_WIDGET_TITLE="Title"
-COM_TJDASHBOARD_FORM_LBL_DASHBOARD_WIDGET_ID="Widget Id"
-COM_TJDASHBOARD_FORM_LBL_DASHBOARD_SIZE="Size"
-COM_TJDASHBOARD_FORM_LBL_DASHBOARD_RENDERER_PLUGIN="Renderer Plugin"
-COM_TJDASHBOARD_FORM_LBL_DASHBOARD_DATA_PLUGIN="Data Plugin"
-COM_TJDASHBOARD_FORM_LBL_DASHBOARD_ID="Dashboard Id"
-
-;
-COM_TJDASHBOARD_SUCCESS_TEXT="Success"
-COM_TJDASHBOARD_ERROR_TEXT_METHOD_NOT_FOUND="Method not exists"
-COM_TJDASHBOARD_ERROR_TEXT_CLASS_NOT_FOUND="Class not exists"
-COM_TJDASHBOARD_ERROR_TEXT_FILE_NOT_FOUND="File not exists"
-COM_TJDASHBOARD_ERROR_TEXT_FOLDER_NOT_FOUND="Folder not exists"
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_WIDGET_ID="Widget Id"
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_SIZE="Size"
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_RENDERER_PLUGIN="Renderer Plugin"
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_DATA_PLUGIN="Data Plugin"
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_ID="Dashboard Id"
+
+;Error For No data/Widget
+COM_TJDASHBOARD_WIDGETS_NOTSHOW_ERROR_MESSAGE="No widgets found to show"
+COM_TJDASHBOARD_WIDGETS_NO_RENDERER_DATA_ERROR_MESSAGE="No data to render"
+COM_TJDASHBOARD_WIDGETS_NO_DATA_ERROR_MESSAGE="No data"
+COM_TJDASHBOARD_NO_DATA_AVAILABLE_MESSAGE="No data Available"
```

Comment:
Above 3 also seem to be duplicates. Can you check which ones are needed and remove unused ones. If all 3 are used see if you can just keep 1.